Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -166,7 +166,9 @@ file that was distributed with this source code.
         {% endif %}
         <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
             {{- widget|raw -}}
-            {{- label is not same as(false) ? label|trans({}, translation_domain) -}}
+            {%- if sonata_admin.options['use_icheck'] -%}&nbsp;{%- endif -%}
```

Comment:
> making an override in CSS implies relying on a class that may change for non-obvious reasons

Yep, valid point.
I tend to think about non-breaking chars as a typographic feature (keeping two related words together on the same line)...
The advantage is that an override can be easily overriden :wink: !
And you can add a nice explainer in your stylesheet:
```css
/**
 * The iCheck checkboxes & radios have 0 margin by default,
 * add some space for the label text.
 */
label > [class^="icheckbox_"],
label > [class^="iradio_"] {
    margin-right: 0.5rem;
}
```